bbq/vm/value_array.go

```diff
@@ -39,4 +39,20 @@ func init() {
 			},
 		),
 	)
+
+	RegisterTypeBoundFunction(
+		commons.TypeQualifierArray,
+		NewNativeFunctionValue(
+			sema.ArrayTypeAppendFunctionName,
+			// TODO:
```

Comment:
Added this and below one to the issue https://github.com/onflow/cadence/issues/3864